i18n/fr/docusaurus-theme-classic/footer.json

```
@@ -0,0 +1,42 @@
+{
+  "link.title.Docs": {
+    "message": "Docs",
+    "description": "The title of the footer links column with title=Docs in the footer"
+  },
+  "link.title.Community": {
+    "message": "Communauté",
+    "description": "The title of the footer links column with title=Community in the footer"
+  },
+  "link.title.More": {
+    "message": "Plus",
+    "description": "The title of the footer links column with title=More in the footer"
+  },
+  "link.item.label.Tutorial": {
+    "message": "Didacticiel",
+    "description": "The label of footer link with label=Tutorial linking to docs/tutorial/welcome-to-redwood"
+  },
+  "link.item.label.Discord": {
+    "message": "Discord",
+    "description": "The label of footer link with label=Discord linking to https://discord.com/invite/redwoodjs"
+  },
+  "link.item.label.Discourse": {
+    "message": "Discours",
```

Let's keep the translation as "Discourse" since it is a proper noun
```suggestion
    "message": "Discourse",
```